example-android/gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,6 @@
+#Fri May 19 11:15:00 EEST 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Might as well use 8.1.1 if you're upgrading.
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
```

---

Actually, why is there a separate gradle wrapper in a sub-folder instead of using the root gradle wrapper? And why does its version now differ?

---

The wrapper in the example-android folder is because of Android Studio. Usually you only import the sub-folder so it needs a separate gradle wrapper